client/src/document/ingredients/browser-compatibility-table/index.tsx

```diff
@@ -103,6 +103,16 @@ export function BrowserCompatibilityTable({
           </tbody>
         </table>
         <Legend compat={data} />
+
+        {/* https://github.com/mdn/yari/issues/1191 */}
+        <div className="hidden">
```

Comment:
I think you wanted this instead?
```suggestion
        <div class="hidden">
```

Reply:
No, this is React.